src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+User:
+  id: -1
```

Comment:
You shouldn't provide some confidential information in public access, read some topics how store secrets in your application